addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -5186,9 +5173,6 @@ registry.layout_column = SnippetOptionWidget.extend({
         }
         newColumnEl.style.gridArea = `${rowStart} / ${columnStart} / ${rowStart + numberRows} / ${columnStart + numberColumns}`;
 
-        // Setting the z-index to the maximum of the grid.
-        gridUtils._setElementToMaxZindex(newColumnEl, rowEl);
-
         // Add the new column and update the grid height.
```

Comment:
You should not remove these lines because without this, when we add a new element with the "Add element" option, it now goes behind the elements already in the grid instead of in front of them and so we don't see it directly anymore (only its overlay) :wink: 
This is also the part that adds the z-index property to these elements and so without that, it is not consistent

Reply:
Done :)